Docker.DotNet/Properties/AssemblyInfo.cs

```diff
@@ -29,5 +29,5 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.16.0.0")]
+[assembly: AssemblyFileVersion("1.16.0.0")]
```

Comment:
@jgarverick hmm not really actually. we're doing that using the nuget pkg version. If you could revert only this file I'd be happy to merge right away.